packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -325,4 +326,30 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       throw detailedError;
     }
   }
+
+  /**
+   * Handles the device type specific logic
+   */
+  private async handleDeviceType(deviceType: string, dn: string): Promise<void> {
```

Comment:
In this method, we simply run a switch case for all possible device types and based on them, we assign the config object `deviceType`.
Improtant thing to note is that for `BROWSER` we **NEED** to do a WebexCallingService re-register as otherwise we have no events sent to mobius (has missed this before).